src/geocel/GeantGeoParams.cc

```diff
@@ -383,6 +383,65 @@ std::vector<inp::Surface> make_inp_surfaces(GeantGeoParams const& geo)
     return result;
 }
 
+//---------------------------------------------------------------------------//
+/*!
+ * Create sensitive detectors input from Geant4 sensitive detectors.
+ */
+std::vector<inp::Detector> make_inp_detectors(GeantGeoParams const& geo)
+{
+    // Process detectors
+    std::vector<inp::Detector> result;
+
+    auto const& vol_labels = geo.impl_volumes();
+
+    // Process each logical volume
+    for (auto iv_id : range(ImplVolumeId{vol_labels.size()}))
+    {
+        auto const& label = vol_labels.at(iv_id);
+        if (label.empty())
+        {
+            // This volume isn't part of the world hierarchy
+            continue;
+        }
+
+        auto vol_id = geo.volume_id(iv_id);
```

Comment:
I think this should work:
```suggestion
        auto vol_id = geo.volume_id(iv_id);
        if (!vol_id)
        {
            // This volume isn't part of the world hierarchy
            continue;
        }
```